client-java/controller/src/main/java/org/evomaster/client/java/controller/internal/EMController.java

```diff
@@ -507,6 +508,8 @@ public Response executeDatabaseCommand(DatabaseCommandDto dto, @Context HttpServ
 
         try {
 
+            sutController.setExecutingInitSql(true);
```

Comment:
hmmm, i have been wondering, is this even necessary? all the SQL commands here are executed in `SqlScriptRunner`, which is not instrumented. so in theory they commands should be not recorded, regardless of this code, isn't it?